libs/fscp/include/fscp/server.hpp

```diff
@@ -1255,7 +1255,7 @@ namespace fscp
 			void do_set_identity(const identity_store&, void_handler_type);
 
 		private:
-			elliptic_curve_list_type get_elliptic_curves_supported(const elliptic_curve_list_type& curves);
+			elliptic_curve_list_type get_supported_curves_elliptic(const elliptic_curve_list_type& curves);
```

Comment:
You have a mismatch: `get_supported_elliptic_curves` <> `get_supported_curves_elliptic`